### ISSUE TITLE
New maintainer for Redmi Note 4/4X

### ISSUE THREAD

**Issue by NRanjan-17** [PixelExperience/official_devices#2681]
## Device you want to apply for

Redmi Note 4/4X

## Device tree sources

https://github.com/NRanjan-17/device_xiaomi_mido
https://github.com/NRanjan-17/kernel_xiaomi_mido
https://github.com/NRanjan-17/vendor_xiaomi_mido

## Number of ROM's you maintain officially

PixysOS

## Any Exceptions/special concern?

No

### Telegram username

@HyperPower17

### GitHub username

NRanjan-17

**Comment by NRanjan-17** [comment]
Please check if I am fit for maintainer or not.
If not, then please tell where I am lagging and how to fix it.
Thank you

**Comment by NRanjan-17** [comment]
An update
Now, I officially maintain PixysOS
Thank You

**Comment by Hlcpereira** [comment]
Hello. Thanks for applying and sorry for the delay in the answer.

Unfortunately we have to reject your application at the moment.

"Why?"
We checked the tree and we found that some commits might've been kanged, but the main reason has to do with a lack of experience for the amount of responsability that comes with been part of the PixelExperience team.

That doesn't mean however that if you apply at a near future you'll be rejected aswell. Feel free to apply whenever you've gathered more experience.

Best regards.
